logs-lib/logs/variables.libsonnet

```diff
@@ -52,8 +52,11 @@ function(
       + [self.regex_search],
 
     queriesSelector:
-      '%s,%s' % [
-        filterSelector,
-        utils.labelsToPromQLSelector(labels),
-      ],
+      std.join(
+        ',',
+        std.filter(function(x) std.length(x) > 0, [
+          filterSelector,
```

Comment:
Do we not want to use the stripped `_filterSelector` here, to avoid adding unnecessary commas?

Reply:
Good point, added.